src/components/CoinListHeader/index.js

```diff
@@ -1,25 +1,129 @@
+              <FaCaretDown />
+            )}
+          </div>
+        </ColumnHeader>
+        <ColumnHeader lg={{ span: 4 }} onClick={() => handleSort("name")}>
+          <div>
+            Name
+            {sortBy !== "name" ? "" : order ? <FaCaretUp /> : <FaCaretDown />}
+          </div>
+        </ColumnHeader>
+        <ColumnHeader
+          lg={{ span: 2 }}
+          onClick={() => handleSort("current_price")}
+        >
+          <div>
+            Price
+            {sortBy !== "current_price" ? (
+              ""
+            ) : order ? (
+              <FaCaretUp />
+            ) : (
+              <FaCaretDown />
+            )}
+          </div>
+        </ColumnHeader>
 
-      <ColumnHeader span={5}>
-        <div>Name</div>
-      </ColumnHeader>
-      <ColumnHeader span={2}>
-        <div>Price</div>
-      </ColumnHeader>
-      <ColumnHeader span={2}>
-        <div>24h %</div>
-      </ColumnHeader>
-    </Row>
-  );
-};
+        <Col lg={{ span: 5 }}>
+          <Row>
+            <ColumnHeader
+              span={8}
+              onClick={() =>
+                handleSort("price_change_percentage_1h_in_currency")
+              }
+            >
+              <div>
+                1h %
+                {sortBy !== "price_change_percentage_1h_in_currency" ? (
+                  ""
+                ) : order ? (
+                  <FaCaretUp />
+                ) : (
+                  <FaCaretDown />
+                )}
+              </div>
+            </ColumnHeader>
+            <ColumnHeader
+              span={8}
+              onClick={() =>
+                handleSort("price_change_percentage_24h_in_currency")
+              }
+            >
+              <div>
+                24h %
+                {sortBy !== "price_change_percentage_24h_in_currency" ? (
+                  ""
+                ) : order ? (
+                  <FaCaretUp />
+                ) : (
+                  <FaCaretDown />
+                )}
+              </div>
+            </ColumnHeader>
+            <ColumnHeader
+              span={8}
+              onClick={() =>
+                handleSort("price_change_percentage_7d_in_currency")
+              }
+            >
+              <div>
+                7d %
+                {sortBy !== "price_change_percentage_7d_in_currency" ? (
+                  ""
+                ) : order ? (
+                  <FaCaretUp />
+                ) : (
+                  <FaCaretDown />
+                )}
+              </div>
+            </ColumnHeader>
+          </Row>
+        </Col>
+
+        <ColumnHeader
+          lg={{ span: 4 }}
+          onClick={() => handleSort("total_volume")}
+        >
+          <div>
+            24h Volume
+            {sortBy !== "total_volume" ? (
+              ""
+            ) : order ? (
+              <FaCaretUp />
+            ) : (
+              <FaCaretDown />
+            )}
+          </div>
+        </ColumnHeader>
```

Comment:
All these Column Header components seem to render a similar structure of component, couls you maybe do a map throug an array and achieve a similar result? (Again not sure)